ibis/expr/types/numeric.py

```diff
@@ -461,6 +461,8 @@ def mean(
         NumericScalar
             The mean of the input expression
         """
+        # TODO(kszucs): remove the alias from the reduction method in favor
+        # of default name generated by ops.Value operations
         return ops.Mean(self, where=where).to_expr().name("mean")
```

Comment:
I tend prefer the default naming scheme here, so `t.float_col.mean()` gets the name `Mean(float_col)` instead of the current `mean`, opinions?

Reply:
Let's remove the current set of default aliases in a follow-up.